SPEC.md

```diff
@@ -201,25 +201,170 @@ Note that logging mechanisms other than stdout and stderr are not required by th
 ### Isolators
 
 Isolators enforce resource constraints rather than namespacing.
-Isolators may be applied to individual applications, to whole containers, or to both.
+Isolators may be scoped to individual applications, to whole containers, or to both.
```

Comment:
how about this, and we can discuss making this information available via the metadata service: "But, an executor MUST make information about which isolators were ignored, enforced or modified to the user."